src/test/java/api/support/fixtures/ItemsFixture.java

```diff
@@ -3,6 +3,8 @@
 import static java.util.function.Function.identity;
 import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
 import static org.folio.circulation.support.JsonPropertyWriter.write;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
```

Comment:
I think we have a consistency problem here and it does *not* have to be addressed by this PR, I just want to note it and see if there is an agreement. Circulation tests import `assertThat` in one of the 3 following ways:
- `org.hamcrest.junit.MatcherAssert.assertThat`
- `org.hamcrest.MatcherAssert.assertThat`
- `org.junit.Assert.assertThat`

This is pretty confusing, since I believe all the implementations are likely equivalent and interchangeable. For example, the `org.junit.Assert.assertThat` method ultimately calls `org.hamcrest.MatcherAssert.assertThat`.

There is even a test that does this:
```java
import static org.hamcrest.junit.MatcherAssert.assertThat;
import static org.junit.Assert.*;
```
Which `assertThat` is used in this case? Spoiler alert, according to Eclipse it uses `org.hamcrest.junit.MatcherAssert.assertThat`. It is pretty obvious here, since the imports are together, but if they were much farther apart, one might not notice that their expected `assertThat` is not imported. Also, we should not be using a glob import here, but that is another problem. As I said, it is probably not a big deal to get one `assertThat` vs another.

We should decide which import to use, whatever is future ready since JUnit5 does not have an `assertThat` assuming we want to use `assertThat` with JUnit5, and make all existing and future tests consistent.

Reply:
@mreno-EBSCO Thank you for this comment.
@mreno-EBSCO @marcjohnson-kint what do you think - which "assertThat" will be better in this PR and everywhere in the system ?

Reply:
That is true, we do have a consistency issue. And it is bigger than this pull request.

@mreno-EBSCO  Do you know which of the aliases is the `right` one to use?

Reply:
@marc-johnson , @mreno-EBSCO The conversation about assertThat dependencies seems to be larger that the scope of this PR.  Therefore, can the PR be merged and further converstation moved to https://issues.folio.org/browse/CIRC-604 ?

Reply:
> @marc-johnson , @mreno-EBSCO The assertThat problem seems to be larger that the scope of this PR. Therefore, can the PR be merged and further converstation moved to https://issues.folio.org/browse/CIRC-604 ?

I think that it can be merged, because it`s already approved, and we created a ticket (CIRC-604) for this issue with "assertThat".